lib/i18n.js

```diff
@@ -99,11 +99,13 @@ export const i18n = {
             };
 
             render() {
-                let {children, ...params} = this.props;
-                return reactjs.createElement('span', {
+                const {children, tagType, props= {}, ...params} = this.props;
+                return reactjs.createElement(tagType || type || 'span', {
                     dangerouslySetInnerHTML: {
                         __html: translator(children, params)
-                    }, key: this.state.systemlocale
+                    },
+                    key: this.state.systemlocale,
+                    ..._.omit(props, [ 'dangerouslySetInnerHTML', 'key' ])
```

Comment:
That `_.omit` can be simply ommited - just pass `...props` first - those two will override the `props` ones.